psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/FactsDeviceData.java

```diff
@@ -23,15 +23,14 @@
  */
 class FactsDeviceData extends AbstractRecordGroup<PsseFacts> {
 
-    private static final String[] FIELD_NAMES_32_33 = {"name", "i", "j", "mode", "pdes", "qdes", "vset", "shmx", "trmx", "vtmn", "vtmx", "vxmx", "imx", "linx",
-        "rmpct", "owner", "set1", "set2", "vsref", "remot", STR_MNAME};
+    private static final String[] FIELD_NAMES_32_33 = {"name", "i", "j", "mode", "pdes", "qdes", "vset", "shmx", "trmx", "vtmn", "vtmx", "vsmx", "imx", "linx", "rmpct", "owner", "set1", "set2", "vsref", "remot", STR_MNAME};
```

Comment:
Can you add a unit test for the `vxmx` -> `vsmx` change?

Reply:
This change has already been checked by the test `ieee14BusCompletedWriteTest`. A bad reference to a PSSE attribute only affects the export process. A badly defined reference causes the attribute not to be exported. As you can see in the differences of the exported file, before the attribute was not exported, `two commas were together`. After the change, the attribute is exported with the default value (1.0).

![image](https://github.com/user-attachments/assets/dc1af402-fca1-4d81-97e3-a39ca1743b8e)

Reply:
Thanks, I missed this one.